kitsune/questions/tests/test_views.py

```diff
@@ -27,7 +27,9 @@
 from kitsune.wiki.tests import DocumentFactory, RevisionFactory
 
 
-class AAQTests(ElasticTestCase):
+# Note:
+# Tests using the ElasticTestCase are not being run bc of this line: `-a '!search_tests'`
+class AAQSearchTests(ElasticTestCase):
```

Comment:
Just an fyi -- I noticed that tests using the `ElasticTestCase` class are excluded when we run `make test`. I also noticed that these tests seem to be failing.